test/models/story_test.rb

```diff
@@ -171,6 +171,14 @@
         story.unpublish!
       end.must_raise(RuntimeError)
     end
+
+    it 'allows the published date to be set via boolean' do
+      story.published = false
+      story.published.must_be_nil
```

Comment:
it may look that way, but I am overriding the `event_attribute` gem default functionality, and that means `published` and `published=` are affected: https://github.com/PRX/cms.prx.org/pull/212/files#diff-8a4ad5774f65a8a217b216fa2038a5bcR169